project-base/storefront/components/Pages/Customer/OrdersContent.tsx

```diff
@@ -131,6 +145,22 @@ export const OrdersContent: FC<OrdersContentProps> = ({ isLoading, breadcrumbs,
                     <Pagination totalCount={totalCount || 0} paginationScrollTargetRef={paginationScrollTargetRef} />
                 </Webline>
             </div>
+            {!!orderForPrefillingUuid && (
+                <Popup onCloseCallback={() => setOrderForPrefillingUuid(undefined)}>
+                    <Heading type="h3">{t('Do you want to add your current cart to the previous order?')}</Heading>
+                    <div className="flex justify-between">
+                        <Button
+                            onClick={() => mergeOrderItemsWithCurrentCart(orderForPrefillingUuid!, false)}
+                            variant="secondary"
+                        >
+                            {t('No')}
+                        </Button>
+                        <Button onClick={() => mergeOrderItemsWithCurrentCart(orderForPrefillingUuid!, true)}>
+                            {t('Yes')}
+                        </Button>
```

Comment:
Marking as non-null assertion doesn't seem to be necessary here.

```suggestion
                        <Button onClick={() => mergeOrderItemsWithCurrentCart(orderForPrefillingUuid, true)}>
                            {t('Yes')}
                        </Button>
```

Reply:
Nice catch, thanks